docs/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### 6.5.0 Features
 
+- `[Searchfield]` Exposed the `collapseSize` setting for use with the IDS Enteprise Searchfield component ([#719](https://github.com/infor-design/enterprise-ng/issues/719))
```

Comment:
I'm never sure when to add these to Fixes or Features, do we have a standard?

Reply:
Some of these are murky , like adding a setting. I guess if its newish its a feature. If its older and missing its a bug. But i wouldn't argue adding it to either one.

Reply:
I'm not sure we do actually.  Looking back on the EP issue #3296, this was actually labeled as a bug, so I wouldn't mind moving it down to Fixes